package.json

```diff
@@ -47,6 +47,6 @@
     "joi": "^10.4.2",
     "knex": "^0.12.9",
     "pg": "^6.2.2",
-    "reflect": "^0.1.3"
+    "rosie": "^1.6.0"
```

Comment:
Because this is being used only in the tests (and not actually in the production code), it should be added to the dev dependencies instead of the normal dependencies. This means running `npm i rosie --save-dev` instead of `npm i rosie --save`.